concordium-std/src/lib.rs

```diff
@@ -17,6 +17,12 @@
 //! Also note that `concordium-std` version 4 only works with `cargo-concordium`
 //! version 2.1+.
 //!
+//! Version 9 deprecates the module [test_infrastructure] in favour of the
```

Comment:
I did not think we'd be bumping a major version for deprecation.

Are there breaking changes now?

Reply:
Right, I was unsure about which version this would be released as, so I just went with `9` or `9.0.0` everywhere so it would be easy to change.

Reply:
And no, there are no breaking changes.